package.json

```diff
@@ -110,6 +111,7 @@
     "@vue/vue3-jest": "^29.2.5",
     "babel-jest": "^29.0.1",
     "babel-loader-exclude-node-modules-except": "^1.2.1",
+    "babel-preset-typescript-vue3": "^2.0.17",
```

Comment:
This preset seems to not be very widely-known and maintained with almost no installations, 5 stars, and rare updates...

Reply:
Well the alternative would be copy-paste the rules from the preset, we only need this for vue styleguideist which still uses webpack+babel

Reply:
Or we get rid of vue-styleguidist and webpack first and migrate to vitepress? But I guess we can also keep it for now and migrate later on.

Reply:
Or even migrate to #174

Reply:
> Or even migrate to #174

Why not, but that looks like a lot more work than vitepress, no?

Reply:
Yes that is what @ShGKme and I fear :sweat_smile: 
But it has some great benefits, as always the problem is missing time.

Reply:
Does Vitepress mean to write all the documentation manually? Event props list...

Reply:
> Does Vitepress mean to write all the documentation manually? Even props list...

No, this will be extracted. @susnux started a first try in #3565, see https://deploy-preview-3565--nextcloud-vue-components.netlify.app
It already kind of works, but needs some adjustements and a fix for the components preview.

Reply:
Yes instead of using vue-styleguideist which uses webpack / vuepress we can use the vue-styleguidist cli tools to extract the documentation as markdown which we then serve using vitepress, see the example:

https://github.com/vue-styleguidist/vue-styleguidist/tree/dev/examples/docgen-vite

Reply:
I dropped that preset and added a simple config.